docs/debugging-tests.md

```diff
@@ -26,13 +26,15 @@ Unfortunately debugging tests in the browser via Node is not great, but they are
 
 ## Debugging in VisualStudioCode
 
-* Open your JavaScript application.
-* Add a `debugger;` statement into the file and line you want to trigger a breakpoint, add a red check on the line number.
+* Open your JavaScript project (either the folder, or go straight for the file you want to debug and open that).
+* Add a `debugger;` statement into the file and line you want to trigger a breakpoint (or add a red check on the line number).
 * Open the debugger panel by clicking on the debugger icon on the left.
-* Choose to add a configuration at the top of the debugger panel.
-* Add the following config to the `launch.json` file which is opened:
+* Select add a configuration at the top of the debugger panel.
+* Click on the cog to select this option and create a config file.
+* Add the following config to the `launch.json` replacing the contents of the `configurations` array:
 
 ```
+// this should be the only element inside the configurations array
```

Comment:
don't need this comment - devs tend to copy everything inside the backticks including comments and they may want additional configurations

Reply:
Fair enough, we wanted to catch devs who just cut and paste the code, but will remove